modules/linear_boltzmann_solvers/solvers/pi_keigen_solver.cc

```diff
@@ -5,8 +5,8 @@
 #include "modules/linear_boltzmann_solvers/discrete_ordinates_problem/discrete_ordinates_problem.h"
 #include "modules/linear_boltzmann_solvers/lbs_problem/lbs_vecops.h"
 #include "modules/linear_boltzmann_solvers/lbs_problem/lbs_compute.h"
-#include "framework/logging/log_exceptions.h"
 #include "framework/logging/log.h"
+#include <stdexcept>
```

Comment:
Should be at the end of the `#include` list

Reply:
Fixed.